pkgs/data/icons/google-cursor/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenvNoCC
+, fetchzip
+, lib
+}:
+
+let
+  colors = [
+    {
+      name = "Black";
+      hash = "sha256-pb2U9j1m8uJaILxUxKqp8q9FGuwzZsQvhPP3bfGZL5I=";
+    }
+    {
+      name = "Blue";
+      hash = "sha256-PmJeGShQLIC7ceRwQvSbphqz19fKptksZeHKi9QSL5Y=";
+    }
+    {
+      name = "Red";
+      hash = "sha256-/X81jLoWaw4UMoDRf1f6oaKKRWexQc4PAACy3doV4Kc=";
+    }
+    {
+      name = "White";
+      hash = "sha256-eT/Zy6O6TBD6G8q/dg+9rNYDHutLLxEY1lvLDP90b+g=";
+    }
+  ];
+in
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "google-cursor";
+  version = "2.0.0";
+
+  sourceRoot = ".";
+  srcs = map
+    (color: (fetchzip {
+      url = "https://github.com/ful1e5/Google_Cursor/releases/download/v${finalAttrs.version}/GoogleDot-${color.name}.tar.gz";
+      name = "GoogleDot-${color.name}";
+      hash = color.hash;
+    }))
+    colors;
+
+  postInstall = ''
+    mkdir -p $out/share/icons
+    cp -r GoogleDot-* $out/share/icons
+  '';
+
+  meta = with lib; {
+    description = "An opensource cursor theme inspired by Google.";
+    homepage = "https://github.com/ful1e5/Google_Cursor";
+    license = licenses.gpl3;
```

Comment:
Oh sure, will do

Reply:
Sorry, could you clarify the difference between only and plus? Here is the license if it matters https://github.com/ful1e5/Google_Cursor/blob/main/LICENSE

Reply:
Oh i think it should be only? https://github.com/ful1e5/Google_Cursor/blob/main/CONTRIBUTING.md#license

Reply:
Should now be fixed.

Reply:
For reference: Upstream, instead of taking the example from the license text and applying it to their code files like the text tells them to - presumably since the source in question mostly consists of graphical data - [has instead opted for filling in the placeholders in the example from the license text](https://github.com/ful1e5/Google_Cursor/blob/ed1ef928e327117b294af66a1a8c68da28e0a8e3/LICENSE#L629-L658)?

They kept the "or any later version" bit there, so we're going with `gpl3Plus` for now? :woman_shrugging:

Reply:
Yeah.